package.json

```diff
@@ -29,6 +29,7 @@
     "axios": "^0.19.0",
     "botbuilder": "4.1.7",
     "botframework-directlinejs": "^0.9.17",
+    "clone": "^2.1.2",
```

Comment:
What is this used for?

Reply:
Nice catch.  Had to install it for some BF stuff that I ended up removing.